mifospay/src/main/res/layout/fragment_merchants.xml

```diff
@@ -12,5 +12,29 @@
         android:layout_gravity="center"
         android:layout_margin="@dimen/marginItemsInSectionLarge"
         android:visibility="gone" />
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:id="@+id/merchant_fragment_layout"
+        android:layout_height="match_parent"
+        android:visibility="invisible"
+        android:orientation="vertical">
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:id="@+id/trusted_merchant_text"
+            android:text="@string/trusted_merchants"
+            android:textSize="15sp"
+            android:layout_marginTop="@dimen/marginBetweenSections"
+            android:layout_marginBottom="@dimen/marginBetweenSections"
+            android:layout_marginStart="10dp"
```

Comment:
You've changed marginLeft, but forgot about marginStart. It should be `@dimen/paddingLayout`, not 10dp